### PR TITLE
Ghseeli/real week 2

### PR DESCRIPTION
**Pull request by ghseeli**
This should not include all the real analysis lectures through January. I still may have to give this another look over myself for little typos, but feel free to start reviewing!

### CODE REVIEW

File: Real Analysis/real-analysis-notes.tex [hunk 1]
```diff
@@ -573,4 +576,371 @@ \subsection{Measurable spaces}
...
+where $\ov{R}$ is a metric space. In fact, we can define the metric
+$\phi: \ov{\R} \to [-1,1]$ as follows \[
+  x \mapsto
+  \begin{cases}
+    \frac{2}{\pi} \arctan x \text{ if } x \in \R \\
+    1 \text{ if } x = \infty \\
+    -1 \text{ if } x = -\infty
+  \end{cases}
+\]
+We can then put a subset topology on $\R \subset \ov{\R}$, but this is
```
Comment (cjl8zf) [inline Real Analysis/real-analysis-notes.tex]: subspace topology ?

File: Real Analysis/real-analysis-notes.tex [hunk 2]
```diff
@@ -573,4 +576,371 @@ \subsection{Measurable spaces}
...
+equivalent to the standard topology on $\R$.
+\begin{example}
+  Take $(a_n)_{n \geq 1}$ with $a_n \in \ov{R}$. Then, $\lim_{n \to \infty} a_n =
+  \infty$ means that, for all $A > 0$, there exists an $N > 0$ such
+  that, for all $n \geq N$, $a_n \geq A$ traditionally, but this same
+  definition would let the limit go to $\infty \in \ov{\R}$ with the
+  new structure.
+\end{example}
+After that digression, let us return to our discussion of measures.
+\begin{defn}
```
Comment (cjl8zf) [inline Real Analysis/real-analysis-notes.tex:617]: maybe change from a a real def to something like, one would like to define this as ___, but this is undefined when uncountable, thus we must define it using the sup.

File: Real Analysis/real-analysis-notes.tex [hunk 3]
```diff
@@ -573,4 +576,371 @@ \subsection{Measurable spaces}
...
+  null. Then,
+  \begin{align*}
+    (E \cup F)^c & = E^c \cap F^c \\
+                 & = E^c \cap \left( (F^c \setminus Z^c) \cup Z^c \right) \\
+    & = \left( E^c \cap (F^c \setminus Z^c) \right) \cup (E \cap Z^c)
+    \\
+    & = \left( E^c \cap (Z \setminus F) \right) \cup (E \cap Z^c)
+  \end{align*}
+  and since $Z \setminus F \subset Z$, we are done.
+\end{proof}
```
Comment (cjl8zf) [inline Real Analysis/real-analysis-notes.tex]: maybe add a remark saying that using the idea of the null sets one may extend the measure beyond the sets it is originally defined?

File: Real Analysis/real-analysis-notes.tex (hunk 1 above)
Comment (ghseeli) [reply Real Analysis/real-analysis-notes.tex]: Yes lol

File: Real Analysis/real-analysis-notes.tex (hunk 2 above)
Comment (ghseeli) [reply Real Analysis/real-analysis-notes.tex:617]: Alright, will do!

**Pull request merged**